session-manager/src/session_manager/session_manager.py

```diff
@@ -0,0 +1,121 @@
+import uuid
+
+
+class SessionManager:
```

Comment:
overall this looks good. It would be good to modify test/chat.py so that it uses the SessionManager as well so that we can see both of them working with it.

Reply:
We agreed to do that in a follow on PR